src/main/java/bisq/core/locale/CurrencyUtil.java

```diff
@@ -95,13 +87,17 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
     public static List<CryptoCurrency> createAllSortedCryptoCurrenciesList() {
         final List<CryptoCurrency> result = new ArrayList<>();
 
+        final ServiceLoader<SpecificAltCoinAddressValidator> loader = ServiceLoader.load(SpecificAltCoinAddressValidator.class);
```

Comment:
Loader is used twice, here and in AltCoinAddressValidator. I'm not sure if it makes sense to introduce separate class like `AlitCoinAddressValidators` to encapsulate this one line.